guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/impl/PullRequestImpl.java

```diff
@@ -0,0 +1,84 @@
+package org.guvnor.structure.repositories.impl;
+
+import org.guvnor.structure.repositories.PullRequest;
+import org.guvnor.structure.repositories.PullRequestStatus;
+
+import static org.uberfire.commons.validation.PortablePreconditions.*;
+
+public class PullRequestImpl implements PullRequest {
```

Comment:
I assume this class is also available client-side (since it's implementation is in ```-api``` module and you use ```PortablePreConditions```. If so, should it not be annotated ```@Portable``` and your ```hashCode()``` implementation (when you write it) should ensure the integer value does not overflow (when ran in JavaScript). See [here](https://github.com/droolsjbpm/kie-wb-common/blob/master/kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-api/src/main/java/org/kie/workbench/common/screens/datasource/management/model/DefInfo.java#L92) for an example.